### PR TITLE
DSERV-131 load ADASTRA asb

### PR DESCRIPTION
**Pull request by shengchd**
Modified asb adapter:
- Create a collection of hyperedge between asb edge and its corresponding cell ontology term
- Refer the motif node in property
- Replaced cell ontology table with pre-annotated ontology ids

### CODE REVIEW

File: data/adapters/adastra_asb_adapter.py [hunk 1]
```diff
@@ -1,96 +1,160 @@
...
+# Example file TF_name@cell_name.tsv (e.g. ATF1_HUMAN@HepG2__hepatoblastoma_.tsv):
 # chr	pos	ID	ref	alt	repeat_type	mean_BAD	mean_SNP_per_segment	total_cover	n_aggregated	es_mean_ref	es_mean_altlogitp_ref	fdrp_bh_ref	logitp_alt	fdrp_bh_alt	motif_log_pref	motif_log_palt	motif_fc	motif_pos	motif_orient	motif_conc	novel
 # chr11	129321262.0 rs10750410	A	G		1.25	518.5	73.0	2.0	-1.508200583122832	1.5220631227173078	0.9999438590195968	1.0	3.776801544248756e-06	0.0024711390339222	2.668977799202377	2.9239362481887974	0.8469536347168959	19	+	No Hit	False
 
 
 class ASB(Adapter):
     # 1-based coordinate system
+    ALLOWED_LABELS = ['asb', 'asb_cell_ontology']
+    ONTOLOGY_PRIORITY_LIST = ['CL:', 'UBERON:', 'CLO:', 'EFO:']
+    CELL_ONTOLOGY_ID_MAPPING_PATH = './samples/allele_specific_binding/ADASTRA_cell_ontologies_mapped_ids.tsv'
```
Comment (pedrohr) [inline data/adapters/adastra_asb_adapter.py:21]: we should move these files to a different folder if they are necessary to load production data. I talked to Mingjie and agreed to create a folder called `data_loading_support_files`

File: data/adapters/adastra_asb_adapter.py [hunk 2]
```diff
@@ -1,96 +1,160 @@
...
+                                'fdrp_bh_alt': row[15],
+                                'cell_name': cell_gtrd_name,
+                                'source_url': 'http://gtrd.biouml.org/#!table/gtrd_current.cells/Details/ID=' + cell_gtrd_id
+                            }
+
+                        json.dump(props, parsed_data_file)
+                        parsed_data_file.write('\n')
+
+        parsed_data_file.close()
+        print
```
Comment (pedrohr) [inline data/adapters/adastra_asb_adapter.py:150]: lost print? :)

File: data/schema-config.yaml [hunk 3]
```diff
@@ -436,28 +436,44 @@ translates to:
     - RO:0002513
     - SIO:010082
 
-asb:
+allele specific binding: ## Need to add acc, filter
```
Comment (pedrohr) [inline data/schema-config.yaml:439]: I'm working on the endpoints, I can add them for you later. We can remove the comment. A few yaml file parsers might not support the ## tags

File: data/adapters/adastra_asb_adapter.py (hunk 2 above)
Comment (shengchd) [reply data/adapters/adastra_asb_adapter.py]: oh forgot to delete!

**Review by pedrohr — APPROVED** [review]

**Review by pedrohr — APPROVED** [review]

**Pull request merged**